rllib/core/rl_trainer/rl_trainer.py

```diff
@@ -205,6 +205,14 @@ def update(self, batch: MultiAgentBatch) -> Mapping[str, Any]:
             self.do_distributed_update(batch)
         return self.compile_results(batch, fwd_out, loss, post_processed_gradients)
 
+    def additional_update(self) -> Mapping[str, Any]:
```

Comment:
should it not take in any parameters? like *args, **kwargs? what should be returned? also provide some more context on where this would get called?

Reply:
fair enough, I thought there wasn't a point to adding args kwargs here because users can just do whatever they want

Reply:
then it should be *args, **kwargs to make it explicit. But the returned value should have some explaination.